### ISSUE TITLE
Mobile device accessibility for the visually impaired: problems mapping and recommendations

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#327]
### Links
- Paper: https://link.springer.com/article/10.1007/s10209-017-0540-1

### Abstract
Mobile devices can be an important ally that improves the quality of life of visually impaired people by permitting greater independence in the execution of certain tasks and facilitating social inclusion. This work presents a systematic review that maps out the accessibility issues that visually impaired people experience when they interact with mobile devices. We identified 68 accessibility problems that were mapped into seven problem groups. This mapping was used to propose 28 recommendations to improve the accessibility of mobile devices. This analysis identified the persistence of certain accessibility problems such as the difficulty of typing on keyboards but also finds the emergence of new challenge such as the failure to recognize gesture-based interactions that demand more extensive training for users. This work is important because it provides an overview of accessibility problems that people with visual disabilities experience with mobile devices and proposes a number of accessibility recommendations to guide future studies. The main contribution of this paper is the mapping of accessibility problems into categories and the development of recommendations for identified problems.

- 視覚障害者がモバイルデバイスに関して不便に感じている点を調査
